xs/Stash.xs

```diff
@@ -1192,17 +1192,12 @@ get(root, ident, ...)
     SV *root
     SV *ident
     CODE:
-    AV *args;
```

Comment:
I can't recall what compiler support is required to declare variables in an else block. Will we be breaking older GCCs if we do this? Maybe we have already...

Reply:
looks like we're already doing this on line 246